configstack/module.go

```diff
@@ -586,7 +587,10 @@ func FindWhereWorkingDirIsIncluded(terragruntOptions *options.TerragruntOptions,
 		}
 		stack, err := FindStackInSubfolders(cfgOptions)
 		if err != nil {
-			return nil, err
+			// loggign error as debug since in some cases stack building may fail because parent files can be designed
+			// to work with relative paths from downstream modules
+			terragruntOptions.Logger.Debugf("Failed to build module stack %v", err)
```

Comment:
Is this the real fix for this issue? Where we _don't_ return an error in this case?